src/platform/plugins/shared/discover/public/application/main/state_management/redux/actions/initialize_session.ts

```diff
@@ -97,32 +106,26 @@ export const initializeSession: InternalStateThunkActionCreator<
     }
 
     if (TABS_ENABLED && !wasTabInitialized) {
-      const tabGlobalStateFromLocalStorage =
-        tabsStorageManager.loadTabGlobalStateFromLocalCache(tabId);
+      const tabInitialGlobalState = tabState.initialGlobalState;
 
-      if (tabGlobalStateFromLocalStorage?.filters) {
-        services.filterManager.setGlobalFilters(cloneDeep(tabGlobalStateFromLocalStorage.filters));
+      if (tabInitialGlobalState?.filters) {
+        services.filterManager.setGlobalFilters(cloneDeep(tabInitialGlobalState.filters));
       }
 
-      if (tabGlobalStateFromLocalStorage?.timeRange) {
-        services.timefilter.setTime(tabGlobalStateFromLocalStorage.timeRange);
+      if (tabInitialGlobalState?.timeRange) {
+        services.timefilter.setTime(tabInitialGlobalState.timeRange);
       }
-      if (tabGlobalStateFromLocalStorage?.refreshInterval) {
-        services.timefilter.setRefreshInterval(tabGlobalStateFromLocalStorage.refreshInterval);
+      if (tabInitialGlobalState?.refreshInterval) {
+        services.timefilter.setRefreshInterval(tabInitialGlobalState.refreshInterval);
       }
 
-      const tabAppStateFromLocalStorage = tabsStorageManager.loadTabAppStateFromLocalCache(tabId);
+      const tabInitialAppState = tabState.initialAppState;
 
-      if (tabAppStateFromLocalStorage) {
-        initialUrlState = tabAppStateFromLocalStorage;
+      if (tabInitialAppState) {
+        urlState = cleanupUrlState(cloneDeep(tabInitialAppState), services.uiSettings, true);
```

Comment:
If this was just for duplication, we could probably just replace it without calling `cleanupUrlState`, but I guess we need to because we've changed the way the tabs storage manager persists the state?